IanGRap/Squish
# Bugfix Player Block Sticking

Contributors: Alfredo, the INTERNET

Goal: There was a bug in the game where players could stick to blocks in the air. If a player jumped next to a tall enough column of blocks and held down the analog stick they would look like they were still falling. This fixes that. 

Changes: 
* Block Physics Material: I changed the Friction setting to Minimum instead of Avg. This means that if two objects have a physics material then it will choose the lowest friction. The player's friction is set to 0. So the friction between blocks shouldn't change but the friction between Blocks and the Player should. 

How to test: 
* Go into development and see how the bug works, just to confirm. 
* Then switch to this branch. Play the normal game mode. You should see that the player doesn't stick to the blocks when they jump and hold down the stick. 
* Make sure to test for a few minutes. Make sure that the blocks doesn't start bugging out from this change.
